srcpkgs/loksh/template

```diff
@@ -10,7 +10,7 @@ maintainer="Renato Aguiar <renato@renatoaguiar.net>"
 license="custom:Public Domain"
 homepage="https://github.com/dimkr/loksh"
 distfiles="https://github.com/dimkr/loksh/releases/download/${version}/loksh-${version}.tar.xz"
-checksum=1a36bafc2bfb9dee11ca65f955528960283ed12f9b00c57c523c49b2a1c91dd5
+checksum=c519ba5d2b2ad699474895775915af8fd8e1ace4972927319f13f7522d5a479f
 register_shell="/bin/loksh"
```

Comment:
please add `/usr/bin/loksh` to the register_shell line, as that can be reported by which depending on the PATH variable.